release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1176,6 +1218,11 @@ In the table below, features are marked with the following statuses:
 |GA
 |GA
 
+|Entitled builds with `SharedSecret` objects
+|-
+|-
+|TP
+
```

Comment:
I would use something like "CSI Volumes in OpenShift Builds". The feature allows for more than entitled builds - for instance, builds can also use Vault secrets if the customer installs the Secret Store CSI driver (third party, not supported by Red Hat).

Reply:
I'm reluctant to describe unsupported third-arty capabilities in our release notes. Please have another look at the way I've reworded the release note for BUILD-274 and let's see if there's another way to describe the flexibility that this new TP feature provides.